src/Haskell/Ide/Engine/Transport/JsonStdio.hs

```diff
@@ -160,11 +160,10 @@ instance A.ToJSON WireRequest where
 
 
 instance A.FromJSON WireRequest where
-    parseJSON (A.Object v) = WireReq <$>
-                           v A..: "cmd" <*>
-                           v A..:? "params" A..!= Map.empty
-    -- A non-Object value is of the wrong type, so fail.
-    parseJSON _          = mzero
+    parseJSON = withObject "WireRequest" $ \v ->
+      WireReq <$>
+      v A..: "cmd" <*>
```

Comment:
worth not importing Aeson qualified here to make this consistent with the other files in the repo?